src/test/java/seedu/finance/logic/commands/CommandTestUtil.java

```diff
@@ -46,7 +46,7 @@
     public static final String CATEGORY_DESC_HUSBAND = " " + PREFIX_CATEGORY + VALID_CATEGORY_HUSBAND;
 
     public static final String INVALID_NAME_DESC = " " + PREFIX_NAME + "James&"; // '&' not allowed in names
-    public static final String INVALID_AMOUNT_DESC = " " + PREFIX_AMOUNT + "*2"; // "*" not allowed in amounts
+    public static final String INVALID_AMOUNT_DESC = " " + PREFIX_AMOUNT + "42"; // '*' not allowed in amount
```

Comment:
I think there may be a missing * in the string

Reply:
Corrected! Thanks for highlighting!